legate/core/corelib.py

```diff
@@ -59,6 +59,7 @@ def get_resource_configuration(self) -> ResourceConfig:
         return config
 
     def destroy(self) -> None:
+        print("destroy corelib")
```

Comment:
Leftover debugging code

Reply:
Yes, I plan to remove it before the merge.